components/Organisms/Pick/NoPickList.tsx

```diff
@@ -3,25 +3,27 @@ import { Box } from 'components/Atoms';
 
 export default function NoPickList() {
   return (
-    <Box
-      position="fixed"
-      top="calc(50% - 110px)"
-      left="0px"
-      width="100%"
-      textAlign="center"
-      fontSize="0"
-    >
-      <Alert width="40px" height="40px" />
+    <>
```

Comment:
```suggestion
    <>
```
위 부분이 없이 하나의 <Box> </Box> 로 될 수 있을 것 같은데 추가하신 이유를 알 수 있을까요??

Reply:
아 습관적으로 <></> 쓰는 버릇이 있어서..! 지금봤네요ㅠㅠ
알려주셔서 감사합니다~ 삭제할게요!ㅎㅎ